### PR TITLE
Add a help page for review app

### PR DESCRIPTION
**Pull request by patjouk**
Ref MozillaFoundation/mofo-devops#600

Add a page with links to all test pages generated by the `load-fake-data` script.
I'm not super happy with the view. I'm sure there's another way to do that 404.

@xmatthewx I think you were the one who asked for this page. Is it what you had in mind? ~I'm also wondering if a drop-down menu with those links wouldn't be a better idea. Sadly I'm not really sure how to do that :(~ I just saw the style guide page: maybe I could use the `btn-normal` or `btn-ghost`?

### CODE REVIEW

**Comment by xmatthewx** [comment]
It's perfect! A list of links is easier to use than a drop-down, and gives viewers a quick glimpse at everything available. 

 - Shorten the main menu item. Maybe `DEV HELP`? All caps can help it stand apart from the others. 
 - Maybe expand the text to "Test pages with placeholder content:"
 - On pages, you could remove the quotes and the word page at the end of each. It's implied. It's totally fine as it is, but this would make it more succinct.  

These are all small things. I think this page will help with everyone's workflow and make it more likely for us to catch bugs!

**Comment by patjouk** [comment]
Those are all great suggestions! Fix on the way :)

File: network-api/networkapi/templates/reviewapp-help.html [hunk 1]
```diff
@@ -9,17 +9,17 @@
...
-          <li><a href="/campaigns/multi-page/">"Mutli page" campaign page</a></li>
-          <li><a href="/opportunity/single/">"Single page" opportunity page</a></li>
-          <li><a href="/opportunity/multi-page/">"Mutli page" opportunity page</a></li>
+          <li><a href="/about/">About</a></li>
+          <li><a href="/styleguide/">Style guide</a></li>
+          <li><a href="/people/">People</a></li>
+          <li><a href="/news/">News</a></li>
+          <li><a href="/initiatives/">Initiatives</a></li>
+          <li><a href="/campaigns/single-page/">Single page campaign</a></li>
+          <li><a href="/campaigns/multi-page/">Mutli page campaign</a></li>
```
Comment (cadecairos) [inline network-api/networkapi/templates/reviewapp-help.html]: "Mutli" -> "Multi"

File: network-api/networkapi/templates/reviewapp-help.html [hunk 2]
```diff
@@ -9,17 +9,17 @@
...
+          <li><a href="/opportunity/single/">Single page opportunity</a></li>
+          <li><a href="/opportunity/multi-page/">Mutli page opportunity</a></li>
```
Comment (cadecairos) [inline network-api/networkapi/templates/reviewapp-help.html]: Same here

**Pull request merged**